app/workers/search/bulk_index_worker.rb

```diff
@@ -10,7 +10,9 @@ def perform(object_class, ids)
         where(id: ids).
         find_each.map(&:serialized_search_hash)
 
-      "Search::#{object_class}".constantize.bulk_index(data_hashes)
+      search_class = object_class.constantize::SEARCH_CLASS
+
+      search_class.bulk_index(data_hashes)
```

Comment:
I made this adjustment to handle articles, comments, and podcast episodes use of `Search::FeedContent` as their class.

Reply:
thank you!!!